### PR TITLE
implemented data generation and DB testing code

### PR DESCRIPTION
**Pull request by John-Michael-Long**
@clfhhc please review this PR

### CODE REVIEW

File: cassandraDB/generateData.js [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+        ok = writer.write(insertLine);
+      }
+      i += 1;
+    } while (i <= entryNumber && ok);
+    if (i <= entryNumber) {
+      writer.once('drain', write);
+    }
+  };
+  write()
+}
```
Comment (clfhhc) [inline cassandraDB/generateData.js:232]: style guide: trailing ';' is recommended

File: cassandraDB/insertToCassandra.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    listingData.description,
+    listingData.location,
+    listingData.reviews_str,
+    listingData.dateSubmitted,
+    listingData.rating,
+    listingData.thumbnailCount,
+    listingData.thumbnailSet,
+    listingData.host_data,
+    listingData.image_data,
+    listingData.reviews_data]
```
Comment (clfhhc) [inline cassandraDB/insertToCassandra.js:38]: style guide: put the trailing bracket to the next line

File: dataGenerationPGQL/imgDataGenerator.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+      let dataString = [
+      thumbnail.entryID, 
+      thumbnail.imageSet, 
+      thumbnail.thumbnailID, 
+      thumbnail.imgFileName, 
+      thumbnail.likes, 
+      thumbnail.submitterID]
+
+      fs.appendFile(fileName, '\n'+JSON.parse(JSON.stringify(dataString.join())), function(err) {
```
Comment (clfhhc) [inline dataGenerationPGQL/imgDataGenerator.js:63]: why would you stringify and parse the string and then stringify this again?

File: dataGenerationPGQL/insertImageDataToPGQL.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    }
+
+    const query = client.query(queryStr, (err, result) => {
+        if(err){
+          console.log('insert err:', err)
+        } else {
+          console.log('saved to DB')
+        }
+      })
+    query.on('end', () => { client.end(); });
```
Comment (clfhhc) [inline dataGenerationPGQL/insertImageDataToPGQL.js:27]: no need to close the client when the server continues running

**Review by clfhhc — APPROVED** [review]
looks good

File: cassandraDB/generateData.js (hunk 1 above)
Comment (John-Michael-Long) [reply cassandraDB/generateData.js:232]: done

File: cassandraDB/insertToCassandra.js (hunk 2 above)
Comment (John-Michael-Long) [reply cassandraDB/insertToCassandra.js:38]: done

File: dataGenerationPGQL/imgDataGenerator.js (hunk 3 above)
Comment (John-Michael-Long) [reply dataGenerationPGQL/imgDataGenerator.js:63]: some reason it makes it work

File: dataGenerationPGQL/insertImageDataToPGQL.js (hunk 4 above)
Comment (John-Michael-Long) [reply dataGenerationPGQL/insertImageDataToPGQL.js:27]: removed line 27

**Comment by John-Michael-Long** [comment]
will implement these changes for next PR

**Pull request merged**